packages/dso-toolkit/components/Voorbeelden/10-Patronen/footer.njk

```diff
@@ -1,18 +1,35 @@
 <footer>
   <div class="row">
-    {%- for column in columns -%}
-      <div class="col-sm-6 col-md-3">
-        <h2>{{ column.label }}</h2>
-        <ul>
-          {%- for item in column.items -%}
-            <li><a href="#">{{ item }}</a></li>
-          {% endfor -%}
-        </ul>
-      </div>
-    {% endfor -%}
     <div class="col-sm-6 col-md-3">
-      <h2>Link list component</h2>
-      {% render '@link-list' %}
+      <h2>Basic link</h2>
+      <ul class="dso-link-list">
```

Comment:
ik ben het ff kwijt, waarom doen we geen `render '@dso-link-list'` meer?

Reply:
Omdat er helemaal geen `yml` gebruikt wordt voor de footer.